Resources/Prototypes/Catalog/VendingMachines/seeds.yml

```diff
@@ -4,3 +4,16 @@
   description: "A vending machine containing Security equipment. A label reads \"SECURITY PERSONNEL ONLY\"."
```

Comment:
guess botanists aren't allowed to use it then

Reply:
funny, this was like this before I even touched it, but a recent PR fixed it so it's fine now